File: Activities/AnonymousInnerClass.java
interface Zombie{
    void think();
}
public class AnonymousInnerClass {
    public static void main(String... args){
        Zombie zombie = new Zombie(){
            @Override
            public void think(){
                System.out.println("Zombie is thinking about brains");
            }
        };

        zombie.think();
    }
}

File: Activities/AnonymousInnerClass.kt
/*
    Created by Jayharron @JayMar on 16/2/2022
 */
interface Zombie{
    fun think()
}
fun main(){

    val zombie : Zombie = object : Zombie{
        override fun think(){
            println("Zombie is thinking about brains")
        }
    }

    zombie.think()
}